server/createPage.js

```diff
@@ -0,0 +1,28 @@
+import {query} from './query.js';
+
+//Book creator endpoint
+
+export const createPage = async (req, res) => {
+  if (!req.body)
+    return res.status(406).send('What are you even trying to do here?');
+
+  const {pageDes} = req.body;
+
+  if (!pageDes) return res.status(406).send('Please enter a page description!');
+
+  let newNumber = 1;
+  // Figure out next page number
+  const [pageNumber] = await query(
```

Comment:
It wouldn't return just the page number. Besides destructuring the array, you could destructure the object in it: `[{page_number}]`. That would be the same as if you used `result` and then did `result[0].page_number`.